api/client/commands.go

```diff
@@ -1289,6 +1296,7 @@ func (cli *DockerCli) CmdImport(args ...string) error {
 
 func (cli *DockerCli) CmdPush(args ...string) error {
 	cmd := cli.Subcmd("push", "NAME[:TAG]", "Push an image or a repository to the registry", true)
+	insecure := cmd.Bool([]string{"-insecure"}, false, "Allow communication with insecure registries")
```

Comment:
This doesn't match documentation. Should this be "-insecure-registry" or should the docs be change to "--insecure"?